platforms/cdh5.12.0/docs/CDH-installation.md

```diff
@@ -562,12 +562,18 @@ com.splicemachine.hbase.BackupEndpointObserver</code>
 
 There are a few configuration modifications you might want to make:
 
+* [Enable automatically restart for HBase service](#enable-automatically-restart) if you want HBase recover automatically after some failures.
```

Comment:
Suggest [Enable automatic restart for HBase service] and missing word "to" in "if you want HBase _to_ recover automatically after some failures."